components/Accordion/Accordion.js

```diff
@@ -45,47 +45,49 @@ function Accordion({ accessibilityId, className, content, hasAnimationOnHover })
   const accordionId = `accordion-control-${accessibilityId}`;
 
   return (
-    <Card
-      className={classNames(styles.Accordion, className)}
-      hasAnimationOnHover={hasAnimationOnHover}
-    >
-      <div className={styles.headingContainer}>{content.headingChildren}</div>
-
-      <button
-        aria-controls={contentId}
-        aria-expanded={isContentVisible}
-        className={styles.accordionToggleButton}
-        data-testid={ACCORDION_TOGGLE_BUTTON}
-        id={accordionId}
-        onClick={toggleAccordionContent}
-        type="button"
-      >
-        {isContentVisible ? (
-          <>
-            <ScreenReaderOnly>{toggleMessages.close}</ScreenReaderOnly>
-            <ChevronDown />
-          </>
-        ) : (
-          <>
-            <ScreenReaderOnly>{toggleMessages.open}</ScreenReaderOnly>
-            <ChevronRight />
-          </>
-        )}
-      </button>
-
-      <section
-        aria-hidden={!isContentVisible}
-        className={classNames(styles.accordionContent, {
-          [styles.visible]: isContentVisible,
-          [styles.invisible]: !isContentVisible,
-        })}
-        data-testid={ACCORDION_CONTENT}
-        id={contentId}
-        style={{ display: isContentVisible ? 'block' : 'none' }}
-      >
-        {content.bodyChildren}
-      </section>
-    </Card>
+    <RadixAccordion.Root type="single" className={classNames(styles.Accordion, className)}>
+      <RadixAccordion.Item value={accordionId}>
+        <RadixAccordion.Header>
+          <RadixAccordion.Trigger
+            className={classNames(styles.headingContainer, {
+              [styles.hover]: hasAnimationOnHover,
+            })}
+            onClick={toggleAccordionContent}
+            aria-controls={contentId}
+            aria-expanded={isContentVisible}
+            data-testid={ACCORDION_TOGGLE_BUTTON}
+            id={accordionId}
+          >
+            <div className={styles.headingContainer}>{content.headingChildren}</div>
+            {isContentVisible ? (
```

Comment:
Radix exposes an HTML attribute on the trigger that will allow us to style the transition rather than maintain our own local state (`isContentVisible`). Check out their docs to see how they do it in their example, which is really similar to ours!

https://user-images.githubusercontent.com/9523719/145086636-afa9e65e-e565-474b-aea1-b3ea3b4e5803.mov

Reply:
Updated